public/app/diarization-timeline.js

```
@@ -0,0 +1,457 @@
+    _render();
+  });
+  ro.observe(Timeline.container);
+}
+
+// ─────────────────────────────────────────────
+// RAF loop
+// ─────────────────────────────────────────────
+
+function _startRAF() {
+  const loop = () => {
+    Timeline.rafId = requestAnimationFrame(loop);
```

ESLint

[error] 'requestAnimationFrame' is not defined.

(no-undef)